src/libraries/System.Net.HttpListener/tests/HttpListenerTests.cs

```diff
@@ -165,10 +165,10 @@ public async Task GetContext_StopIsCalled_GetContextUnblocked()
             var listener = listenerFactory.GetListener();
             listener.Start();
             var listenerTask = Task.Run(() => Assert.Throws<HttpListenerException>(() => listener.GetContext()));
-            await Task.Delay(1000).TimeoutAfter(10000); // Wait for listenerTask to call GetContext.
+            await Task.Delay(1000); // Wait for listenerTask to call GetContext.
```

Comment:
this is a bug fix? Why would anyone want to timeout the `Delay`?

Reply:
Yeah, this didn't make sense :)